### DISCUSSION TITLE
VictorySong and LoserSong

### DISCUSSION THREAD

**Discussion by sommo** [micbay/RaceTimerAndController#14] [General]
Hi @micbay ,
could be nice to have the ability to have a single victory song for the first place finisher and a loser song for the last place.

thank you!

**Comment by micbay** [comment]
With [version 2.0+](https://github.com/micbay/RaceTimerAndController/releases/tag/v2.0.0), you can now configure the songs to be associated with finishing place instead of racer names.

By default, songs are associated with racer names, however setting `SONGS_BY_PLACE` to `true`, the controller will associate the songs array index, with racer finishing places, and when the 1st racer finishes, the song in index 1 of the `RACER_SONGS_LIST` will play. Subsequently, when the 2nd racer finishes, the song in index 2 will play, and so forth.

If you want only a song for the first place finisher, put empty song data in the other places.

**Comment by sommo** [comment]
If you want only a song for the first place finisher, put empty song data in the other places.
=> that's thank you!
